torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1,3 +1,4 @@
+#include <aten/src/ATen/Context.h>
```

Comment:
Note that I just removed the prim::None operator, everything else is clang format